### PR TITLE
Added config modal to support number of messages and dev/prod mode feature

### PR DESCRIPTION
**Pull request by aditygrg2**
Related to Issue #24 

Continuation to PR #25. Sorry for closing earlier PR's.

### CODE REVIEW

**Comment by aditygrg2** [comment]
@glaslos This PR is updated. Sorry for two closed PR's.

**Comment by glaslos** [comment]
I'll have a look later. Be aware, that you can always fix issues in your PR via git, there should be no reasons for deleting a PR.

File: src/App.svelte [hunk 1]
```diff
@@ -19,18 +18,70 @@
...
+	$: if (messages.length > noOfMessages) {
 		messages = messages.slice(1);
 	}
 
-	let content: messageType;
-	let filter: string;
-
+	$: if(isDevOn){
+		test();
+	} else{
```
Comment (glaslos) [inline src/App.svelte:36]: Formatting

File: src/App.svelte [hunk 2]
```diff
@@ -49,39 +100,53 @@
...
 
-	function dial() {
-		let conn: WebSocket;
+	function dial(conn: WebSocket) {
 		try {
+			if(!isDevOn)
 			conn = new WebSocket(`wss://${location.host}/subscribe`);
 		} catch (e) {
-			conn = new WebSocket(`ws://${location.host}/subscribe`);
+			conn = new WebSocket(`wss://${location.host}/subscribe`);
```
Comment (glaslos) [inline src/App.svelte:108]: Here try to connect without SSL

File: src/App.svelte [hunk 3]
```diff
@@ -113,6 +180,22 @@
 	<input bind:value={filter} placeholder="Filter destination port" />
 	<button on:click={filterMessages}>Apply</button>
 	<span>Port number and '&&' to concat.</span>
+	<backdrop id="backdrop" on:click={modalHandler} style="visibility: hidden;"></backdrop>
```
Comment (glaslos) [inline src/App.svelte:183]: Why not use the Svelte modal? https://svelte.dev/examples/modal

**Review by glaslos — CHANGES_REQUESTED** [review]

**Pull request closed without merging**